opm/autodiff/SimulatorFullyImplicitBlackoilOutput.cpp

```diff
@@ -296,8 +296,21 @@ namespace Opm
             }
             else {
                 // just write the data to disk
-                writeTimeStepSerial( timer, state, wellState, cellData, substep );
+                int err = 0;
+                try {
+                    writeTimeStepSerial( timer, state, wellState, cellData, substep );
+                    MPI_Bcast(&err, 1, MPI_INT, 0, MPI_COMM_WORLD);
+                } catch (std::runtime_error& msg) {
```

Comment:
pushed a rewrite. if you prefer that one, we use that one. different projects have different rules and i have been used to the rule 'if you shall rethrow it shall be in the catch block'. i still haven't gotten hold of the comm though, it seems to be hiding very well.